test/decorators.test.ts

```diff
@@ -99,4 +100,20 @@ describe('Request decorators', () => {
 
     expect(retry.statusCode).toEqual(401)
   })
+
+  test('logIn allow to assign a serializeUser value to a request.session.passport property directly', async () => {
+    const { server } = getConfiguredFastifyTestServer()
+    server.post('/force-login', async (request: any, reply) => {
+      await request.logIn('force logged in user')
+      void reply.send(JSON.parse(request.session.passport))
+    })
+
+    const login = await server.inject({
+      method: 'POST',
+      url: '/force-login',
+    })
+
+    expect(login.statusCode).toEqual(200)
+    expect(login.body).toEqual('force logged in user')
+  })
```

Comment:
I feel kinda sad that this is the only test that actually exercises the new session lib. What do you think about refactoring some of the more basic tests into a reusable suite, and then apply that suite twice to different base configurations that use the different session libraries?

Reply:
Ok, so do we need to refactor all tests for @fastify/session or support two sets of tests for two session managers?